modules/catalog/src/test/java/org/apache/ignite/internal/catalog/storage/CatalogEntrySerializationTest.java

```diff
@@ -446,12 +446,12 @@ private static CatalogSortedIndexDescriptor newSortedIndexDescriptor(String name
         CatalogIndexColumnDescriptor idxCol4 = new CatalogIndexColumnDescriptor("C4", CatalogColumnCollation.ASC_NULLS_LAST);
 
         return new CatalogSortedIndexDescriptor(
-                1, name, 12, false, CatalogIndexStatus.AVAILABLE, List.of(idxCol1, idxCol2, idxCol3, idxCol4));
+                1, name, 12, false, CatalogIndexStatus.AVAILABLE, List.of(idxCol1, idxCol2, idxCol3, idxCol4), true);
     }
 
     private static CatalogHashIndexDescriptor newHashIndexDescriptor(String name) {
         return new CatalogHashIndexDescriptor(
-                1, name, 12, true, CatalogIndexStatus.REGISTERED, List.of("C1", "C2"));
+                1, name, 12, true, CatalogIndexStatus.REGISTERED, List.of("C1", "C2"), true);
```

Comment:
why `true`?

Reply:
Because `false` is the default value and I wanted to test, that serialization works correctly.